src/docs/development/data-and-backend/state-mgmt/options.md

```diff
@@ -98,6 +98,14 @@ A family of stream/observable based patterns.
 * [Reactive Programming - Streams - BLoC - Practical Use Cases][],
   by Didier Boelens
 
+## GetIt
+
+A service locator based state management approach that doesn't need a `BuildContext`.
+
+* [get_it package][], The service locator. It can also be used together with BloCs.
```

Comment:
[nit] Consistent capitalization would help here. If the package's name is "GetIt", I'd go with "GetIt package" (similar to "Provider package" and "Flutter Redux package" elsewhere on this page). Conversely, if the preferred stylization is `get_it`, then please update the heading of this section.

This applies to the other bullet points as well.

Reply:
ok